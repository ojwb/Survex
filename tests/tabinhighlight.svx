; pos=fail warn=0 error=1
*fix	1	reference	0	0	0	bad	tab
