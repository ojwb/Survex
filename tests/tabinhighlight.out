./tabinhighlight.svx:2:24: error: End of line not blank
 *fix 1 reference 0 0 0 bad tab
                        ^~~~~~~

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 1 survey station, joined by 0 legs.
There are 0 loops.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m

There were 0 warning(s) and 1 error(s) - no output files produced.
